docs/integrations/sources/redshift.md

```diff
@@ -54,6 +54,7 @@ All Redshift connections are encrypted using SSL
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.3.8   | 2022-01-26 | [9744](https://github.com/airbytehq/airbyte/pull/9721) | Implement getPrivilegesTableForCurrentUser |
```

Comment:
The changlog can probably be more descriptive from the perspective of the user instead of the code change. Maybe something like:

```suggestion
List only the tables on which the user has SELECT permissions.
```